buildspec/linuxE2ETests.yml

```
@@ -37,7 +37,7 @@ phases:
         commands:
             - export HOME=/home/codebuild-user
             # Ignore failure until throttling issues are fixed.
-            - xvfb-run npm run testE2E
+            - xvfb-run npm run testE2E; PREVIOUS_TEST_EXIT_CODE=$? npm run mergeReports
```

can avoid this env var by passing the exit code directly as an arg
```suggestion
            - xvfb-run npm run testE2E; npm run mergeReports -- "$?"
```

package.ts shows how to get script args: https://github.com/aws/aws-toolkit-vscode/blob/5b5df855b2cc2f12ea8e1298e1ebee5cfe6b598a/scripts/package.ts#L40-L42